modules/ldap-auto-syncing.adoc

```diff
@@ -0,0 +1,246 @@
+// Module included in the following assemblies:
+//
+// * authentication/ldap-syncing.adoc
+
+[id="ldap-auto-syncing_{context}"]
+= Automatically syncing LDAP groups
+
+You can automatically sync LDAP groups on a periodic basis by configuring a cron job.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have configured an LDAP identity provider (IDP), which involved creating an LDAP secret and config map.
+
+// TODO Any other prereqs? what about anything else w/ the sync stuff, like a config file, etc? Looks like they embed it in the config map
+
+.Procedure
+
+. Create a namespace where the cron job will run:
++
+[source,terminal]
+----
+$ oc create namespace ldap-sync <1>
+----
+<1> This procedure uses a namespace called `ldap-sync`.
+
+. Copy the secret and config map that you created when configuring the LDAP identity provider to this new namespace.
++
+The secret and config map exist in the `openshift-config` namespace and must be copied to the new `ldap-sync` namespace.
++
+// TODO: Provide actual instructions on how to copy, or just leave it like this?
+
+. Define a service account:
++
+.Example `ldap-sync-service-account.yaml`
+[source,yaml]
+----
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: ldap-group-syncer
+  namespace: ldap-sync
+  labels:
+    app: cronjob-ldap-group-sync
```

Comment:
@stlaz This label is used on each object definition. 

Is this required? If so, can you tell me why? And then if so, is it also required on the config map and secret that were created when setting up the LDAP IDP?

Reply:
it's not required, I think the author of the original post wanted to be able to remove all of these resources at once and so they labeled them as such.

You don't need that now because you're using a separate NS and so you can just remove the whole NS instead.

Reply:
Okay great, I'll remove it